### PR TITLE
VR-4198 overwrite check test cases fix

### PR DESCRIPTION
**Pull request by evgeniilezhnin**

### CODE REVIEW

File: backend/src/test/java/ai/verta/modeldb/ExperimentRunTest.java [hunk 1]
```diff
@@ -8489,29 +8490,36 @@ public void logGetVersioningExperimentRunCreateTest()
...
+    }
+
+    try {
+      experimentRunServiceStub.logVersionedInput(logVersionedInputFail);
+      fail();
+    } catch (StatusRuntimeException e) {
+      Status status = Status.fromThrowable(e);
+      LOGGER.warn("Error Code : " + status.getCode() + " Description : " + status.getDescription());
+      assertEquals(Status.ALREADY_EXISTS.getCode(), status.getCode());
+    }
```
Comment (anandjakhaniya) [inline backend/src/test/java/ai/verta/modeldb/ExperimentRunTest.java]: I think this is redundant. this should be a single case.

**Review by anandjakhaniya — CHANGES_REQUESTED** [review]
Looks like two same cases added in the test class. added comment on it

File: backend/src/test/java/ai/verta/modeldb/ExperimentRunTest.java (hunk 1 above)
Comment (evgeniilezhnin) [reply backend/src/test/java/ai/verta/modeldb/ExperimentRunTest.java]: removed it

**Review by anandjakhaniya — APPROVED** [review]
LGTM

**Pull request merged**